ocamlformat.opam

```diff
@@ -18,7 +18,7 @@ dev-repo: "git+https://github.com/ocaml-ppx/ocamlformat.git"
 url { archive: "https://github.com/ocaml-ppx/ocamlformat/archive/%%VERSION%%.tar.gz" }
 license: "MIT"
 build: [
-  ["ocaml" "tools/gen_version.mlt" "src/Version.ml" version] {pinned}
+  ["dune" "subst"] {pinned}
```

Comment:
The only `%%VERSION%%` left is in the `.opam` file, is it useful to subtitute it ?
That URL (`url.archive`) wouldn't work anyway, what about removing it from the opam file ?

Reply:
̀dune subst` will add `(version ...)` to `dune-project`, so it's useful even without placeholders. I saw the url params in opam files, that's dodgy indeed, I'll see what i can do.

Reply:
I see.